program-analysis/echidna/advanced/state-network-forking.md

```diff
@@ -0,0 +1,90 @@
+# On-chain fuzzing with state forking
+
+**Table of contents:**
+
+- [On-chain fuzzing with state forking](#on-chain-fuzzing-with-state-forking)
+  - [Example](#example)
+  - [Corpus and RPC cache](#corpus-and-rpc-cache)
+  - [Coverage and Etherscan integration](#coverage-and-etherscan-integration)
+
+## Example
+
+One of the most anticipated features of Echidna 2.1.0 is the state network forking. This means that Echidna can run starting with an existing blockchain state provided by an external RPC service (Infura, Alchemy, local node, etc).
```

Comment:
I think we should reword this, and remove the "most anticipated features" part

Reply:
We should also have the introduction in an introduction section, and not in the example

Reply:
I reworked this as introduction and example, please take a look